control/veda_deliver.py

```diff
@@ -184,6 +139,29 @@ def run(self):
             if self.video_query.provider == TranscriptProvider.CIELO24:
                 self.cielo24_transcription_flow()
 
+    def hls_run(self):
+        """
+        Get information about encode for URL validation/record
+
+        """
+        self.video_query = Video.objects.filter(edx_id=self.veda_id).latest()
```

Comment:
.latest() could break.

Reply:
How?

Reply:
Both latest and get will break potentially, if there's an ID mismatch, but that's ok in this instance, because it's a celery process and it will "fail" and pick up the next task.